alphabase/psm_reader/psm_reader.py

```diff
@@ -251,115 +233,34 @@ def import_file(self, _file: str) -> pd.DataFrame:
-                min_rt = max(min_rt, self._min_irt_value)
-                max_rt = min(max_rt, self._max_irt_value)
-
-            elif not self._min_max_rt_norm:
-                min_rt = 0
-
-            self._psm_df[PsmDfCols.RT_NORM] = (
-                (self._psm_df[PsmDfCols.RT] - min_rt) / (max_rt - min_rt)
-            ).clip(0, 1)
-
-    def normalize_rt_by_raw_name(self) -> None:
-        """Normalize RT by raw name."""
-        if PsmDfCols.RT not in self._psm_df.columns:
-            return
-        if PsmDfCols.RT_NORM not in self._psm_df.columns:
-            self._normalize_rt()
-
-        if PsmDfCols.RAW_NAME not in self._psm_df.columns:
-            return
-        for _, df_group in self._psm_df.groupby(PsmDfCols.RAW_NAME):
-            self._psm_df.loc[df_group.index, PsmDfCols.RT_NORM] = (
-                df_group[PsmDfCols.RT_NORM] / df_group[PsmDfCols.RT_NORM].max()
-            )
-
     def _load_file(self, filename: str) -> pd.DataFrame:
         """Load PSM file into a dataframe.
 
         Different search engines may store PSMs in different ways: tsv, csv, HDF, XML, ...
         This default implementation works for tsv and csv files and thus covers many readers.
-
-        Parameters
-        ----------
-        filename : str
-            psm filename
-
-        Returns
-        -------
-        pd.DataFrame
-            psm file as dataframe
-
         """
         sep = _get_delimiter(filename)
         return pd.read_csv(filename, sep=sep, keep_default_na=False)
 
-    def _find_mapped_columns(self, df: pd.DataFrame) -> Dict[str, str]:
-        """Determine the mapping of AlphaBase columns to the columns in the given DataFrame.
+    def _get_mod_seq_column(self, origin_df: pd.DataFrame) -> Optional[str]:
+        """Get the first column from `_mod_seq_columns` that is a column of `df`."""
+        for mod_seq_col in self._mod_seq_columns:
+            if mod_seq_col in origin_df.columns:
+                return mod_seq_col
+        return None
+        # TODO: warn if there's more
 
-        For each AlphaBase column name, check if the corresponding search engine-specific
-        name is in the DataFrame columns. If it is, add it to the mapping.
-        If the searchengine-specific name is a list, use the first column name in the list.
-        """
-        mapped_columns = {}
-        for col_alphabase, col_other in self.column_mapping.items():
-            if isinstance(col_other, str):
-                if col_other in df.columns:
-                    mapped_columns[col_alphabase] = col_other
-            elif isinstance(col_other, (list, tuple)):
-                for other_col in col_other:
-                    if other_col in df.columns:
-                        mapped_columns[col_alphabase] = other_col
-                        break
-                        # TODO: warn if there's more
-        return mapped_columns
+    def _pre_process(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
naming question: the original dataframe (the one loaded from disk) is sometimes referred to as `df` sometimes as `origin_df` .. any preferences? (or a third option?)

Reply:
origin_df is better